akka-stream-tests/src/test/scala/akka/stream/io/DeprecatedTlsSpec.scala

```diff
@@ -389,11 +389,11 @@ class DeprecatedTlsSpec extends StreamSpec(DeprecatedTlsSpec.configOverrides) wi
             .collect { case SessionBytes(_, b) => b }
             .scan(ByteString.empty)(_ ++ _)
             .filter(_.nonEmpty)
-            .via(new Timeout(6.seconds))
+            .via(new Timeout(10.seconds))
```

Comment:
Do we use dilation in those tests? If yes we could add it as well.

Reply:
I don't think we ever use dilation in the CI where this is failing, but perhaps good nonetheless